### PR TITLE
[hi] Localize api-eviction.md

### PR DESCRIPTION
**Pull request by bishal7679**
Signed-off-by: Bishal Das <bishalhnj127@gmail.com>

Localizes `content/en/docs/reference/glossary/api-eviction.md` to Hindi `content/hi/docs/reference/glossary/api-eviction.md`

Fixes :- #35938
Mentioned issue :- #35600

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Reusable

### CODE REVIEW

File: content/hi/docs/reference/glossary/api-eviction.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+short_description: >
+  एपीआई द्वारा शुरू की गईी निष्कासन वह प्रक्रिया है जिसके द्वारा आप एविक्शन एपीआई का उपयोग एक निष्कासन वस्तु बनाने के लिए करते हैं जो ग्रेसफुल पॉड टर्मिनेशन को ट्रिगर करता है।
+aka:
+tags:
+  - operation
+---
+
+एपीआई द्वारा शुरू किया गया निष्कासन वह प्रक्रिया है जिसके द्वारा [निष्कासन एपीआई](/docs/reference/generated/kubernetes-api/{{<param "version">}}/#create-eviction-pod-v1-core) का उपयोग एक `निष्कासन` वस्तु बनाने के लिए किया जाता हैं जो ग्रेसफुल पॉड समापन को ट्रिगर करता है।
+
+<!--more-->
```
Comment (Aut0R3V) [inline content/hi/docs/reference/glossary/api-eviction.md:15]: Great work. Were you able to figure out the reason for the errors?

File: content/hi/docs/reference/glossary/api-eviction.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+date: 2021-04-27
+full_link: /docs/concepts/scheduling-eviction/api-eviction/
+short_description: >
+  एपीआई द्वारा शुरू की गईी निष्कासन वह प्रक्रिया है जिसके द्वारा आप एविक्शन एपीआई का उपयोग एक निष्कासन वस्तु बनाने के लिए करते हैं जो ग्रेसफुल पॉड टर्मिनेशन को ट्रिगर करता है।
+aka:
+tags:
+  - operation
+---
+
+एपीआई द्वारा शुरू किया गया निष्कासन वह प्रक्रिया है जिसके द्वारा [निष्कासन एपीआई](/docs/reference/generated/kubernetes-api/{{<param "version">}}/#create-eviction-pod-v1-core) का उपयोग एक `निष्कासन` वस्तु बनाने के लिए किया जाता हैं जो ग्रेसफुल पॉड समापन को ट्रिगर करता है।
```
Comment (Aut0R3V) [inline content/hi/docs/reference/glossary/api-eviction.md:13]: I believe the text in blue should be in English.

File: content/hi/docs/reference/glossary/api-eviction.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+<!--more-->
+
+आप सीधे निष्कासन एपीआई को कॉल करके निष्कासन का अनुरोध कर सकते हैं कुबे-एपीआईसर्वर के क्लाइंट का उपयोगा करके, जैसे `kubectl drain` कमांड। जब एक `निष्कासन` वस्तु बनाया जाता है, तो एपीआई सर्वर पॉड को समाप्त कर देता है।
+
+एपीआई द्वारा शुरू किया गया निष्कासन आपके कॉन्फ़िगर किए गए [`PodDisruptionBudgets`](/docs/tasks/run-application/configure-pdb/) और [`terminationGracePeriodSeconds`](/docs/concepts/workloads/pods/pod-lifecycle#pod-termination) का सम्मान करते हैं।
+
+एपीआई द्वारा शुरू की गईी निष्कासन [node-pressure eviction](/docs/concepts/scheduling-eviction/node-pressure-eviction/) के समान नहीं है।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/api-eviction.md:21]:
```suggestion
API द्वारा शुरू किया गया निष्कासन [नोड-प्रेशर एविक्शन](/docs/concepts/scheduling-eviction/node-pressure-eviction/) से अलग है।
```

File: content/hi/docs/reference/glossary/api-eviction.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+title: एपीआई द्वारा शुरू किया गया निष्कासन (API-initiated eviction)
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/api-eviction.md:2]:
```suggestion
title: API द्वारा शुरू किया गया निष्कासन (API-initiated eviction)
```

**Review by Babapool — COMMENTED** [review]
Some suggestions from myside. Hope you find them useful. Thanks:

**Comment by bishal7679** [comment]
/cc @divya-mohan0209 
PHAL! and left suggestions if have any. Then I'll go for squashing all the commits!

File: content/hi/docs/reference/glossary/api-eviction.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
+---
+title: API द्वारा शुरू किया गया एविक्शन (API-initiated eviction)
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/api-eviction.md:2]: Isn't निष्कासन more commonly used for Eviction in Hindi?

File: content/hi/docs/reference/glossary/api-eviction.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+id: api-eviction
+date: 2021-04-27
+full_link: /docs/concepts/scheduling-eviction/api-eviction/
+short_description: >
+  API द्वारा शुरू किया गया एविक्शन वह प्रक्रिया है जिसके द्वारा आप एविक्शन (Eviction) API का उपयोग एक एविक्शन ऑब्जेक्ट बनाने के लिए करते हैं जो ग्रेसफुल पॉड टर्मिनेशन को ट्रिगर करता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/api-eviction.md:7]: ट्रिगर => चालू कर देना

**Review by divya-mohan0209 — COMMENTED** [review]
Left a couple of suggestions, PTAL

**Comment by Babapool** [comment]
Hi @bishal7679, just wanted to confirm since there is no update on PR for a long time, are you actively working on this PR? Thanks

**Comment by bishal7679** [comment]
Hey @Babapool I've just committed the suggestion given by @divya-mohan0209
PTAL!

**Pull request closed without merging**